coral/package.json

```diff
@@ -28,22 +28,22 @@
     ]
   },
   "dependencies": {
-    "@aivenio/design-system": "^18.4.3",
+    "@aivenio/design-system": "^19.0.1",
     "@hookform/resolvers": "^2.9.10",
-    "lodash": "^4.17.21",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.38.0",
     "react-router-dom": "^6.4.2",
     "zod": "^3.19.1"
   },
   "devDependencies": {
+    "@originjs/vite-plugin-commonjs": "^1.0.3",
```

Comment:
For me it is not clear why we need to dependency. Is it a leftover from some lodash commonJS prototyping?

Reply:
🙈  yes, it's a leftover. I did go over all changes again before submitting the PR, but sees like I overlooked this one. Thanks for noticing! 

Also good reminder to create [this issue](https://github.com/orgs/aiven/projects/10)!